locales/en-US/app.ftl

```diff
@@ -182,6 +182,9 @@ newsletterFooterBody = Find out about new experiments and see test results for e
 newsletterFooterSuccessHeader = Thanks!
 newsletterFooterSuccessBody = If you haven't previously confirmed a subscription to a Mozilla-related newsletter you may have to do so. Please check your inbox or your spam filter for an email from us.
 
+localeWarningTitle = This experiment is only available in English.
```

Comment:
For now yeah, we'll be blocklisting some experiments in German and grantlisting others to German. I built this to handle to/from any language though, so the copy should probably support it.